[2024-08-02T14:30:43.137+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-02T14:30:43.147+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline_supermarket_2.copy_table_dim_aisles manual__2024-08-02T07:09:40.143577+00:00 [queued]>
[2024-08-02T14:30:43.153+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline_supermarket_2.copy_table_dim_aisles manual__2024-08-02T07:09:40.143577+00:00 [queued]>
[2024-08-02T14:30:43.154+0700] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-08-02T14:30:43.179+0700] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): copy_table_dim_aisles> on 2024-08-02 07:09:40.143577+00:00
[2024-08-02T14:30:43.182+0700] {standard_task_runner.py:64} INFO - Started process 16520 to run task
[2024-08-02T14:30:43.184+0700] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline_supermarket_2', 'copy_table_dim_aisles', 'manual__2024-08-02T07:09:40.143577+00:00', '--job-id', '143', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_supermarket_2.py', '--cfg-path', '/tmp/tmp8ccrzul9']
[2024-08-02T14:30:43.185+0700] {standard_task_runner.py:91} INFO - Job 143: Subtask copy_table_dim_aisles
[2024-08-02T14:30:43.223+0700] {task_command.py:426} INFO - Running <TaskInstance: data_pipeline_supermarket_2.copy_table_dim_aisles manual__2024-08-02T07:09:40.143577+00:00 [running]> on host DESKTOP-RHVHV7A.
[2024-08-02T14:30:43.277+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_pipeline_supermarket_2' AIRFLOW_CTX_TASK_ID='copy_table_dim_aisles' AIRFLOW_CTX_EXECUTION_DATE='2024-08-02T07:09:40.143577+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-02T07:09:40.143577+00:00'
[2024-08-02T14:30:43.277+0700] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-02T14:30:43.286+0700] {sql.py:282} INFO - Executing: 
        COPY dim_aisles TO '/mnt/f/AllCode/Data Engineer/dim_aisles.csv' CSV HEADER;
        
[2024-08-02T14:30:43.290+0700] {base.py:84} INFO - Using connection ID 'postgres_supermarket' for task execution.
[2024-08-02T14:30:43.306+0700] {base.py:84} INFO - Using connection ID 'postgres_supermarket' for task execution.
[2024-08-02T14:30:43.440+0700] {sql.py:495} INFO - Running statement: 
        COPY dim_aisles TO '/mnt/f/AllCode/Data Engineer/dim_aisles.csv' CSV HEADER;
        , parameters: None
[2024-08-02T14:30:43.442+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-02T14:30:43.442+0700] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/common/sql/operators/sql.py", line 288, in execute
    output = hook.run(
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/common/sql/hooks/sql.py", line 442, in run
    self._run_command(cur, sql_statement, parameters)
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/common/sql/hooks/sql.py", line 500, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedTable: relation "dim_aisles" does not exist

[2024-08-02T14:30:43.445+0700] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=data_pipeline_supermarket_2, task_id=copy_table_dim_aisles, run_id=manual__2024-08-02T07:09:40.143577+00:00, execution_date=20240802T070940, start_date=20240802T073043, end_date=20240802T073043
[2024-08-02T14:30:43.466+0700] {standard_task_runner.py:110} ERROR - Failed to execute job 143 for task copy_table_dim_aisles (relation "dim_aisles" does not exist
; 16520)
[2024-08-02T14:30:43.477+0700] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-08-02T14:30:43.485+0700] {logging_mixin.py:188} WARNING - /usr/local/lib/python3.10/dist-packages/airflow/models/baseoperator.py:1297 AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `airflow.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
[2024-08-02T14:30:43.494+0700] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-02T14:30:43.509+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
